tests/integration/construct_component_resource_options/go/go.mod

```diff
@@ -1,17 +1,17 @@
 module github.com/pulumi/pulumi/tests/construct_component_resource_options
 
-go 1.18
+go 1.20
 
 replace github.com/pulumi/pulumi/sdk/v3 => ../../../../sdk
 
-require github.com/pulumi/pulumi/sdk/v3 v3.62.0
+require github.com/pulumi/pulumi/sdk/v3 v3.85.1-0.20230926151835-2b44cf6ec1be
```

Comment:
Could this be `v3.87.0`?

Reply:
Same--this is `make tidy`'s doing--I'm inclined to take what it gives us